schemas/manifest.fbs

```diff
@@ -95,6 +95,9 @@ table ManifestV1 {
     // needs this to determine whether it's safe to drop duplicate key writes. If a recent snapshot
     // still references an older version of a key, it should not be dropped.
     recent_snapshot_min_seq: ulong;
+
+    // a mapping from sequence number to the corresponding system clock timestamp
+    seq_tracker: SequenceTracker (required);
```

Comment:
just realized this is backwards incompatible. I'll make it optional or figure out if there's a way in the serde layer to have a default if it's not there